instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -18,6 +18,8 @@ def call(_worker_class, job, _queue, _redis_pool)
               attributes = {
                 SemanticConventions::Trace::MESSAGING_SYSTEM => 'sidekiq',
                 'messaging.sidekiq.job_class' => job['wrapped']&.to_s || job['class'],
+                'messaging.sidekiq.retry' => job['retry'].to_s,
+                'messaging.sidekiq.args' => job['args'].join(', '),
```

Comment:
Please consider amending your specific domain attributes to the job in custom code as part of your application. 

Including arguments in an unstructured value makes instrumentations susceptible to leaking PII and are incredibly difficult to redact in post processing.  

```suggestion
```